engine/model_contact_test.go

```diff
@@ -102,3 +96,62 @@ func Test_getContact(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func Test_getContacts(t *testing.T) {
+	t.Run("status 'not confirmed'", func(t *testing.T) {
+		ctx, client, deferMe := CreateTestSQLiteClient(t, false, false, withTaskManagerMockup())
+		defer deferMe()
+
+		var metadata *Metadata
+
+		dbConditions := make(map[string]interface{})
+
+		var queryParams *datastore.QueryParams
+
+		(dbConditions)[xPubIDField] = xPubID
+		(dbConditions)[contactStatus] = notConfirmed
```

Comment:
Is it necessary to put the `dbConditions` variable into parenthesis?

Reply:
Changed. Thanks

Reply:
During review I've noticed that you haven'tt changed it in all places where the `(dbConditions)[` occurs. Please adjust.